docs/guides/issue_triage.md

```diff
@@ -0,0 +1,135 @@
+# Project issues
```

Comment:
General thing - I'm trying to do a bit of a docs cleanup for the mpsg and one of the things we want to do is to move stuff out of the guides folder because it tends to becomes a dumping ground. Would you mind moving this maybe into the ci-cd folder and changing the name of that to be something like gihub maintenance or something other such thing?

Reply:
Moved it to top level. I could find no good placement for this :(